terraform/modules/iam_role/concourse_worker/role.tf

```diff
@@ -43,7 +43,8 @@ module "concourse" {
         "s3:PutObject"
```

Comment:
This is a versioned bucket, I think we need to allow:
s3:PutObject
s3:PutObjectAcl
s3:GetObject
s3:ListBucket
s3:GetBucketVersioning
s3:GetObjectVersion
s3:ListBucketVersions
s3:PutObjectVersionAcl